### DISCUSSION TITLE
Welcome to EssentialsX Discussions!

### DISCUSSION THREAD

**Discussion by mdcfe** [EssentialsX/Essentials#3780] [General]
## 👋 Welcome!
Welcome to the EssentialsX Discussions forum! This is the forum for all things EssentialsX.

Check out some of our categories:
* [#️⃣ **General**](https://github.com/EssentialsX/Essentials/discussions/categories/general): Chat with other members of the EssentialsX community!
* [💡 **Ideas and Feature Suggestions**](https://github.com/EssentialsX/Essentials/discussions/categories/ideas-and-feature-suggestions): Share and discuss ideas for new features!
* [🙏 **Q&A**](https://github.com/EssentialsX/Essentials/discussions/categories/q-a): Ask questions and get help with EssentialsX!
* [🙌 **Show and tell**](https://github.com/EssentialsX/Essentials/discussions/categories/show-and-tell): Show off your creations and share setups built with EssentialsX!

We have the following guidelines in place:
* Be respectful and courteous to others.
* Don't advertise.
  * Linking to your work in the context of a conversation is okay.
  * Posting links to servers/plugins out of nowhere is not.
* Be sure to post in relevant categories and topics.
  * If you're posting a feature request or support request, be sure to search before creating a new thread - you might find a relevant discussion or solution in an existing thread already.

If you've got any feedback on the forum, feel free to reply to this thread!

**Comment by Chew** [comment]
Exciting stuff. Can't wait to use this more!

**Comment by LaxWasHere** [comment]
\o/

**Comment by Turbotailz** [comment]
neat

**Comment by ArcticArua** [comment]
Hello, I am currently experiencing issues with EssentialsXSpawn. EssentialsX works perfectly fine, but when I try to add the latest dev build into my server Plugin Files, it gives me a message saying that the server is outdated, when I'm using Paper Version 1.16.4. I am not sure what the issue is, but my friend also uses Essentials and he also has EssentialsX and EssentialsXSpawn to work together. I been trying to figure this problem out for about 2 weeks, yet to never find a solution. I am open to screen sharing, screen shots, screen recording, and even 1 on 1 talking,
![image](https://user-images.githubusercontent.com/75389340/104821914-0dcb5c00-57f4-11eb-9dd3-8c8ac270f7ab.png)
![image](https://user-images.githubusercontent.com/75389340/104821920-19b71e00-57f4-11eb-9571-33cd9b53e790.png)

**Reply by JasonHorkles** [reply]
answered in the discord
